VideoWeb/VideoWeb/ClientApp/src/app/testing/mocks/data/conference-test-data.ts

```diff
@@ -440,9 +440,23 @@ export class ConferenceTestData {
             linked_participants: []
         });
 
+        const participant4 = new ParticipantResponseVho({
```

Comment:
I think we should in general move away from using this test data and build the data we need in the tests to keep them relevant. But it's fine here.